exporter.go

```diff
@@ -25,7 +25,7 @@ type WmiCollector struct {
 }
 
 const (
-	defaultCollectors = "cpu,cs,logical_disk,net,os,system"
+	defaultCollectors = "cpu,cs,logical_disk,net,os,service,system"
```

Comment:
Let's enable it for now. I have the feeling this is still a relative lightweight monitoring solution compared to the alternatives I looked at.